### PR TITLE
Unify controller worker num param `threadiness` to `workers`

### PR DESCRIPTION
**Pull request by astraw99**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
In the controller `Run` method, there are different worker num params, this PR unify them to 1 consistent name `workers`, then the code is more precisely to read.

#### Special notes for your reviewer:
The `v1` alias is automatically added by IDE(Goland), to better differentiate from other v1 packages.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go [hunk 1]
```diff
@@ -153,8 +153,8 @@ func (c *autoRegisterController) Run(threadiness int, stopCh <-chan struct{}) {
 		}
 	}
 
-	// start up your worker threads based on threadiness.  Some controllers have multiple kinds of workers
-	for i := 0; i < threadiness; i++ {
+	// start up your worker threads based on workers.  Some controllers have multiple kinds of workers
+	for i := 0; i < workers; i++ {
```
Comment (leilajal) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go:157]: Just commenting out of curiosity, if the worker is already a thread and some controllers have multiple threads, why do we need to name it as a worker?

Comment (astraw99) [reply staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go:157]: The thread number of majority controllers are named `workers`, so just unify them for code consistency.

**Review by leilajal — APPROVED** [review]

**Comment by astraw99** [comment]
@jsafrane PTAL, thanks.

**Pull request merged**